ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/ApiPartyManagementService.scala

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.services.admin
+
+import akka.stream.Materializer
+import com.daml.ledger.participant.state.index.v2.IndexPartyManagementService
+import com.daml.ledger.participant.state.v2.{PartyAllocationResult, WriteService}
+import com.digitalasset.grpc.adapter.ExecutionSequencerFactory
+import com.digitalasset.ledger.api.v1.admin.party_management_service.PartyManagementServiceGrpc.PartyManagementService
+import com.digitalasset.ledger.api.v1.admin.party_management_service._
+import com.digitalasset.platform.api.grpc.GrpcApiService
+import com.digitalasset.platform.common.util.{DirectExecutionContext => DE}
+import com.digitalasset.platform.server.api.validation.ErrorFactories
+import io.grpc.{BindableService, ServerServiceDefinition}
+import org.slf4j.LoggerFactory
+import scalaz.syntax.tag._
+
+import scala.compat.java8.FutureConverters
+import scala.concurrent.{ExecutionContext, Future}
+
+class ApiPartyManagementService private (
+    readBackend: IndexPartyManagementService,
```

Comment:
This is not necessarily a read backend. In other services we called this like the service, e.g. `transactionService: IndexTransactionService`.